SU-HSDO/suhumsci
# Arch 306 search

# READY FOR REVIEW

# Summary
- Use search api to allow more control over search results and querying.

# Needed By (Date)
- August 1st

# Urgency
- Mild

# Steps to Test

1. checkout this branch
1. `composer update --prefer-source`
1. `blt drupal:sync --site=archaeology` (it will delete custom configs, this is be normal)
1. validate there is still a search icon on the top of all the pages.
1. visit the page `/admin/config/search/search-api/index/default_index` and click on "index now" to index nodes
1. go to the home page and search for something
1. validate stuff displays as expected
1. search for a course title that has multiple nodes. like `Independent Study`
1. validate only 1 "independent study" course result displays.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
